CHANGELOG.md

```diff
@@ -15,6 +15,159 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See the [Babel Changelog](https://github.com/babel/babel/blob/master/CHANGELOG.md) for the pre-6.8.0 version Changelog.
 
+## 6.16.0 (2017-02-22)
+
+### :rocket: New Feature
+
+***Estree*** compatibility as plugin ([#277](https://github.com/babel/babylon/pull/277)) (Daniel Tschinder)
+
+We finally introduce a new compatibility layer for estree. The new plugin `estree` can be enabled and puts the parser into
+estree-compatible mode. In this mode the parser will output an AST that is compliant to the specs of [estree](https://github.com/estree/estree/)
+
+We highly recommend everyone who uses babylon outside of babel to use this plugin. Why? It will be a lot easier for you to switch between different estree compatible parsers. We so far tested several projects with different parsers and exchanged their parser to babylon and in most cases it was support easy to replace. (Some other estree-compatible es6-parsers include acorn, esprima, espree, flow-parser, ...) 
```

Comment:
> ~~(Some other estree-compatible es6-parsers include acorn, esprima, espree, flow-parser, ...)~~

This doesn't need to be parenthesized, so something like:

Some other estree-compatible parsers include acorn, esprima, espree, flow-parser, etc.

Also, "estree compatible" or "estree-compatible" should be used consistently.